.travis.yml

```diff
@@ -7,9 +7,12 @@ python:
   - "3.7"
 install:
   - pip install -e './[dev]'
+  - pip install codecov
 before_script:
   - coderedcms start testproject
 # command to run tests
 script:
-  - python testproject/manage.py test coderedcms --settings=coderedcms.tests.settings
-  - flake8 coderedcms testproject
+  - coverage run testproject/manage.py test coderedcms --settings=coderedcms.tests.settings
+  - flake8 coderedcms testproject --exit-zero
+after_success:
+  - codecov
```

Comment:
Is after success mean if there were not test/flake errors? Or does it always run assuming the script didn't bomb out?

Reply:
It won't run if the tests fail.  flake errors don't cause issues.

Reply:
Looks good. Reminder to remove the `--exit-zero` flag in the future once we get the code cleaned and flaked.